### PR TITLE
[post] Google SRE 阅读笔记(1)-监控

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <ce.gao@outlook.com>

## Post info

- Title: Google SRE 阅读笔记(1)-监控
- Author: @gaocegege
- License: default

## Related posts

[1] Ce Gao, [Google SRE 阅读笔记(1)-监控](http://gaocegege.com/Blog/%E9%98%85%E8%AF%BB/sre-0)

### CODE REVIEW

**Comment by gaocegege** [comment]
PTAL

**Comment by arrowrowe** [comment]
:sheep:

File: source/google_sre_阅读笔记(1)-监控.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## 本文介绍
+
+这篇博客是系列文章中的第一篇，主要分享在阅读这本书时的一些感想。这本书在我看来更加适合在分布式领域或者在运维领域工作的工程师阅读，对于一个还在念书，没有完整接触过分布式系统实现的新手来说，有些过早了。因此就当是抛砖引玉，随便写写吧。
+
+这次关注的是书中的第六章，分布式系统的监控。
+
+## 关于作者
+
+第六章的作者是 [Rob Ewaschuk](https://www.linkedin.com/in/robewaschuk)。作者主要工作的领域是分布式存储，而且在自我介绍中写道自己在谷歌干的很过瘾，16-17年是不打算换工作的。O'Reilly 摘录了他在 SRE 一书中关于分布式监控的部分，做了一本电子书 [Monitoring Distributed Systems](http://www.oreilly.com/webops-perf/free/monitoring-distributed-systems.csp)。
```
Comment (at15) [inline source/google_sre_阅读笔记(1)-监控.md:27]: lol 又是这种下载要邮箱的，得弄个专门的邮箱了.....

**Comment by at15** [comment]
btw: is the branch name a typo? `src-0` not `sre-0`?

**Comment by at15** [comment]
LGTM

**Comment by at15** [comment]
You forgot to build the html .... @gaocegege

**Comment by gaocegege** [comment]
Yeah, it's a typo.

**Comment by gaocegege** [comment]
I forgot to pre-deploy, PTAL again, sorry.

@at15 @arrowrowe

**Comment by arrowrowe** [comment]
:sheep: ship it!

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/9/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

**Comment by at15** [comment]
:sheep: Roast the mie!

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/9/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

**Comment by ComMouse** [comment]
Seems that we can do continuous deploy in Travis

**Comment by at15** [comment]
@ComMouse yeah, and that would make PR more clear

**Comment by gaocegege** [comment]
Awesome idea. @ComMouse

**Pull request merged**